shared/index.cjsx

```diff
@@ -41,6 +41,7 @@ module.exports = {
 
   FreeResponse:           require './src/components/exercise/free-response'
   GetPositionMixin:       require './src/components/get-position-mixin'
+  Join:                   require './src/components/enroll/join'
   ScrollToMixin:          require './src/components/scroll-to-mixin'
```

Comment:
Exporting the component as just `Join` seems too generic to me.  Maybe it could be called `NewEnrollment` or `JoinCourse`?  There's probably something better than either of those, but you get the idea.  The thing that it's exported as here doesn't actually have to be the same as the component's `displayName`